spec/models/slide_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe 'Slide' do
+  before do
+    AWSConfig.configure do |config|
+      config.region = 'ap-northeast-1'
+      config.aws_access_id = 'aws_access_id'
+      config.aws_secret_key = 'aws_secret_key'
+      config.bucket_name = 'my-bucket'
+      config.image_bucket_name = 'my-image-bucket'
+      config.sqs_url = 'https://www.ryuzee.com'
+      config.use_s3_static_hosting = '0'
+      config.cdn_base_url = ''
+    end
+  end
+
+  describe 'thumbnail_url' do
+    it 'url is valid' do
+      FactoryGirl.create(:slide)
+      key = Slide.find(1).key
+      expect(Slide.find(1).thumbnail_url).to eq("https://my-image-bucket.s3-ap-northeast-1.amazonaws.com/#{key}/thumbnail.jpg")
+    end
+  end
+
+  describe 'transcript_url' do
+    it 'url is valid' do
+      FactoryGirl.create(:slide)
+      key = Slide.find(1).key
+      expect(Slide.find(1).transcript_url).to eq("https://my-image-bucket.s3-ap-northeast-1.amazonaws.com/#{key}/transcript.txt")
+    end
+  end
+
+  describe 'page_list_url' do
+    it 'url is valid' do
+      FactoryGirl.create(:slide)
+      key = Slide.find(1).key
+      expect(Slide.find(1).page_list_url).to eq("https://my-image-bucket.s3-ap-northeast-1.amazonaws.com/#{key}/list.json")
+    end
+  end
+
+  describe 'page_list' do
+    it 'returns page list as json' do
+      FactoryGirl.create(:slide)
+      url = Slide.find(1).page_list_url
+      stub_request(:any, url).to_return(
+        :body => {"test" => "ok"}.to_json,
```

Comment:
__[rubocop]__
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)
* Style/SpaceInsideHashLiteralBraces: Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceInsideHashLiteralBraces: Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)